### DISCUSSION TITLE
Welcome to obs-vnc Discussions!

### DISCUSSION THREAD

**Discussion by norihiro** [norihiro/obs-vnc#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peterfnet** [comment]
Great work; works like a champ! Using to record a Rockwell HMI for later troubleshooting

**Reply by norihiro** [reply]
I appreciate your comment. It's interesting to see how the plugin is used.

**Comment by satishkumarm7** [comment]
Hi Team,
I would like to thank for such great support in getting the vnc added as source. Can you please share steps how it works and if possible some sample video tutorials for adding the vnc viewer please. as i couldnt add vnc viewer in windows machine.

Regards
Sam

**Reply by norihiro** [reply]
Hi,
The procedure to add the source on OBS Studio is created.
https://github.com/norihiro/obs-vnc/wiki/Setup-VNC-Source
I'm sorry that I'm not so good at making videos so that usually I don't make a tutorials.
